src/dashboards/components/DashboardPage.tsx

```diff
@@ -114,7 +114,8 @@ class DashboardPage extends Component<Props> {
 
   private get pageTitle(): string {
     const {dashboard} = this.props
-    const title = dashboard && dashboard.name ? dashboard.name : 'Loading...'
+    const title =
+      dashboard && dashboard.name ? dashboard.name : 'Name this Dashboard'
```

Comment:
```suggestion
      dashboard?.name ?? 'Name this Dashboard'
```

Reply:
Just a heads up, this will change the meaning of this expression and I'm not sure we want that. Currently, if `dashboard.name` is falsy, like a `0` or `''`, it'll revert to `Name this Dashboard`. If we use the nullish coalescing, it'll only fail on `undefined` or `null`.

```js
const dashboard = {name: ''}
const title = dashboard && dashboard.name ? dashboard.name : 'Name this Dashboard'
// title: Name this Dashboard
```

```js
const dashboard = {name: ''}
const title = dashboard?.name ?? 'Name this Dashboard'
// title:""
```

Reply:
If a user explicitly sets the dashboard name to an empty string, shouldn't that be allowed?